src/components/forms/power-vs/PowerVsCloudConnectionForm.js

```diff
@@ -0,0 +1,142 @@
+            name="pi_cloud_connection_speed"
+            handleInputChange={this.handleInputChange}
+            className="fieldWidth"
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseToggle
+            id={this.props.data.name + "-cloud-connect-global-routing"}
+            defaultToggled={this.state.pi_cloud_connection_global_routing}
+            labelText="Enable Global Routing"
+            onToggle={() =>
+              this.handleToggle("pi_cloud_connection_global_routing")
+            }
+            className="fieldWidth"
+          />
+          <IcseToggle
+            id={this.props.data.name + "-cloud-connect-metered"}
+            defaultToggled={this.state.pi_cloud_connection_metered}
+            labelText="Enable Metered Connection"
+            onToggle={() => this.handleToggle("pi_cloud_connection_metered")}
+            className="fieldWidth"
+          />
+          <IcseToggle
+            id={this.props.data.name + "-cloud-connect-transit-enabled"}
+            defaultToggled={this.state.pi_cloud_connection_transit_enabled}
+            labelText="Enable Transit Gateway"
+            onToggle={() =>
+              this.handleToggle("pi_cloud_connection_transit_enabled")
+            }
+            className="fieldWidth"
+          />
+        </IcseFormGroup>
+        {this.state.pi_cloud_connection_transit_enabled && (
+          <IcseFormGroup>
+            <IcseMultiSelect
+              className="fieldWidthSmaller"
+              id={this.props.data.name + "-cloud-connect-transit-gw"}
+              titleText="Transit Gateways"
+              items={this.props.transitGatewayList}
+              onChange={(value) => {
+                this.handleMultiSelectChange(
+                  "transit_gateways",
+                  value.selectedItems,
+                );
+              }}
+              initialSelectedItems={this.state.transit_gateways}
+              invalid={this.state.transit_gateways.length === 0}
+              invalidText="Select at least one transit gateway"
+            />
+          </IcseFormGroup>
+        )}
+      </>
+    );
+  }
+}
+
+PowerVsCloudConnectionForm.defaultProps = {
+  data: {
+    name: "",
+    pi_cloud_connection_speed: null, //[50, 100, 200, 500, 1000, 2000, 5000, 10000],
+    pi_cloud_connection_global_routing: false,
+    pi_cloud_connection_metered: false,
+    pi_cloud_connection_transit_enabled: false,
+    transit_gateways: [],
+  },
+  isModal: false,
+  transitGatewayList: [],
+};
+
+PowerVsCloudConnectionForm.propTypes = {
+  data: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    pi_cloud_connection_speed: PropTypes.number,
+    pi_cloud_connection_global_routing: PropTypes.bool.isRequired,
+    pi_cloud_connection_metered: PropTypes.bool.isRequired,
+    pi_cloud_connection_transit_enabled: PropTypes.bool.isRequired,
+    transit_gateways: PropTypes.arrayOf(PropTypes.string),
+  }),
+  isModal: PropTypes.bool.isRequired,
```

Comment:
do we need isModal? It doesn't seem to be used anywhere

Reply:
Jen told me she had an issue with a different form when she didn't include it

Reply:
@Pseusco change the return statement to return `""` instead of component when `isModal` is true, needed as this will be hidden when creating a power vs workspace but shown otherwise